src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -512,6 +516,28 @@ To rename the `_astro` path, specify a new directory in `build.assets`.
 }
 ```
 
+<p>
+**If use `Record<string, string>`**
+</p>
+
+Applicable to multiple CDN scenarios. If you static file use different CDN, you can configure it like the example below.
```

Comment:
Since this is documenting new behaviour, we will need to double check with core for accuracy!